ds3-sdk/src/main/java/com/spectralogic/ds3client/helpers/WriteJobImpl.java

```diff
@@ -218,14 +243,18 @@ public void transferItem(final Ds3Client client, final BulkObject ds3Object)
             client.putObject(createRequest(ds3Object));
         }
 
-        private PutObjectRequest createRequest(final BulkObject ds3Object) {
+        private PutObjectRequest createRequest(final BulkObject ds3Object) throws IOException {
+
+
+            final SeekableByteChannel channel = jobState.getChannel(ds3Object.getName(), ds3Object.getOffset(), ds3Object.getLength());
```

Comment:
Indentation is right, but it has extra newlines for no reason, I'm removing them.